boot/init/tasks/auto_resize.rb

```diff
@@ -6,27 +6,69 @@ def initialize(device, type: )
     @device = device
     @type = type
     add_dependency(:Devices, @device)
+    add_dependency(:Mount, "/sys")
+  end
+
+  # Computes whether a filesystem needs to be expanded.
+  # This way the long-running tasks happen only once.
+  # TODO: parse more than ext filesystems
```

Comment:
This TODO is fine as *anyway* it didn't support anything else than ext beforehand.